chunky/src/java/se/llbit/chunky/renderer/scene/Sky.java

```diff
@@ -681,7 +681,13 @@ public void importFromJson(JsonObject json) {
     skyExposure = json.get("skyExposure").doubleValue(skyExposure);
     skyLightModifier = json.get("skyLight").doubleValue(skyLightModifier);
     apparentSkyLightModifier = json.get("apparentSkyLight").doubleValue(apparentSkyLightModifier);
-    mode = SkyMode.get(json.get("mode").stringValue(mode.name()));
+    if (!(Objects.equals(json.get("mode").stringValue(mode.name()), "SKYMAP_PANORAMIC") || Objects.equals(json.get("mode").stringValue(mode.name()), "SKYMAP_SPHERICAL"))) {
```

Comment:
What's the benefit of using `Objects.equals` over the `.equals` method?

Reply:
Objects.equals first does a reference equality `==`, then checks for null and then does `.equals`
Though in this case they can never be `==` as (if I'm reading the brackets correctly) one is a string literal,
so `"SKYMAP_PANORAMIC".equals(json.get("mode").stringValue(mode.name()))` would probably be preferred

Reply:
I have set it to be `json.get("mode").stringValue(mode.name()).equals("SKYMAP_PANORAMIC")`.